.github/workflows/pull-request.yml

```diff
@@ -28,9 +28,8 @@ jobs:
       - name: Update npm to latest
         run: npm i --prefer-online --no-fund --no-audit -g npm@latest
       - run: npm -v
-      - name: Install deps
-        run: npm i -D @commitlint/cli @commitlint/config-conventional
```

Comment:
is the idea here that we would put the commitlint packages in our dev deps ahead of time? i think this is just a one-off install to run those checks.. entirely possible we could do this with a fancy `npm exec` command too